review-tools/gitaddrev

```diff
@@ -138,13 +138,13 @@ foreach (@ARGV) {
         $WHAT = 'tools';
         $min_authors = 1;
         # openssl/tools is governed by OTC
-        $min_otc = 2;
+        $min_otc = 0;
```

Comment:
Is this correct? I'm not necessarily against this change but previously only OTC could commit to this repo. Now any committer can. I'm not sure if that was the intent behind the bylaws change which I think was primarily about the main source repo - and I suppose other repos using the same rules as main source.

Reply:
I think its in keeping with the letter of they bylaws.  As to the spirit, I don't think I can say.  I'd look to other committers/OTC members  on that

Reply:
This is the otc/tools repo in the primary GHE server. IMO this is wrong. The bylaws are about the primary openssl/openssl codebase but can be probably extended to all the openssl/<something> repos.

Please revert this hunk.

Reply:
> I think its in keeping with the letter of they bylaws

I don't think its in keeping with the letter either. It depends on what you think the scope of that sentence is about. But the policy in general talks about what committers can do, and committers are defined to be "contributors with commit access to the OpenSSL source code repository." - which I think is fairly unambiguously openssl/openssl. Other repos have different review and approval criteria which this policy does not cover.

Reply:
so, whats the right change here?  I can agree that min_otc can/should be 2 for the tools repo, and 0 for openssl, what about perftools/installer/etc?

Reply:
I think we can agree that perftools, installer and fuzz-corpora repos should have _similar_ rules as the main source code.

Reply:
Strictly speaking the committer policy that was changed *only* applies to openssl/openssl. However, I think the intent for installer was that it should have the same review requirements as openssl/openssl (although I don't know if that was documented anywhere....possibly in some meeting minutes somewhere). I don't know what the right answer for fuzz-corpora is - probably it needs separate discussion.

Reply:
So can we agree here that we should limit this change to the default case, as the other repos are explicitly called out?  Or do we need to bring this up at OTC?

Reply:
I think we limit it to just the main repo for now. All other repos require a separate agreement as to what the correct review criteria are.